### ISSUE TITLE
Implement MKOB's code sender window

### ISSUE THREAD

**Issue by leskerr** [MorseKOB/PyKOB#78]
Enable keyboard sending as well as the file menu to import text to be sent.

**Comment by AESilky** [comment]
This seems like a duplicate of #55.

#55 is entered as a 'Bug' because it was assumed that the MKOB GUI Application was complete. Is it preferred to use this issue rather than the 'bug' issue?

#55 is already in the 'todo' state.

**Comment by leskerr** [comment]
Oops! I forgot about #55. Sorry! Can we close #55 and go with this one? Or somehow link the two issues?

As you've probably guessed by now, the MKOB GUI app is woefully incomplete.

**Comment by AESilky** [comment]
I defer to you to use the one you prefer - I don't know that we want both open. We can close one of them and have it refer to the other for a trace-history.

I opened the earlier issues one day when I opened both 2.5 and 4.0 and tried to do the same thing on each. For some of the 4.0 issues (opened as 'bugs') I included step-by-step instructions on what I did. For those, I would like to keep those steps - either by keeping the initial issue, or by copying to a new 'feature' issue. I'm also fine with dropping the 'bug' label and changing it to 'enhancement' (or we can create a new label of 'to-be-implemented' (or similar).
